llvm/lib/Target/NVPTX/NVPTXISelDAGToDAG.cpp

```diff
@@ -2095,6 +2168,8 @@ bool NVPTXDAGToDAGISel::tryStoreRetval(SDNode *N) {
   for (unsigned i = 0; i < NumElts; ++i)
     Ops.push_back(N->getOperand(i + 2));
   Ops.push_back(CurDAG->getTargetConstant(OffsetVal, DL, MVT::i32));
+  Ops.push_back(PRED(0));
```

Comment:
Nit: we could fold all those `push_back` into `Ops.append({....})`

Reply:
done